build.sbt

```diff
@@ -2,22 +2,24 @@ name := "play-mockws"
 
 scalaVersion := "2.11.8"
 
-crossScalaVersions := Seq("2.11.8")
+crossScalaVersions := Seq("2.12.1")
```

Comment:
Just tested this with `+publishLocal`: I needed to put both in both scala versions: `crossScalaVersions := Seq("2.11.8", "2.12.1")`.